big_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -194,6 +194,9 @@ end_per_testcase(delete_old_users, Config) ->
         end, Users),
     escalus:end_per_testcase(delete_old_users, Config);
 end_per_testcase(CaseName, Config) ->
+    %% Because kick_session fails with unexprected stanza received:
```

Comment:
`unexpected`

Add `TODO: Remove when escalus learns how to automatically deal with 'unavailable' stanzas on client stop.`